### DISCUSSION TITLE
Run on docker

### DISCUSSION THREAD

**Discussion by nicknesh** [magiblot/turbo#64] [Show and tell]
It's just a suggestion. I like your work! 25 years ago I mostly wrote code for turbovision. The first thing I tried was running on docker.
Example Dockerfile:

```
FROM alpine as build
RUN apk add linux-headers ncurses-dev build-base cmake git
ARG nproc=13
RUN git clone --recursive https://github.com/magiblot/turbo.git && cd turbo && \
    cmake . -DCMAKE_BUILD_TYPE=Release && cmake --build . -- -j $nproc

FROM alpine
COPY --from=build /turbo/turbo /usr/local/bin/turbo
RUN apk add ncurses-dev
CMD ["/usr/local/bin/turbo"]

```
build:
`docker build  -t turbo-alpine .`

The docker image is about 16MB. I tried on Fedora, Ubuntu and Debian.
I run image:
`docker run -v /data:/data -it --rm turbo-alpine`

**Comment by magiblot** [comment]
Hi, Nick! I'm glad you like it. Thanks for your comment.

As I understand, this is not a proposal to make any changes to the project, right? In this case, I will convert this issue into a discussion.

**Comment by nicknesh** [comment]
Hi, Convert to discussion, please!
